frontend/components/common.tsx

```diff
@@ -182,7 +156,7 @@ export function _renderControls() {
           <Account />
         </header>
 
-        <TwapPrice />
+        {account && <TwapPrice />}
```

Comment:
It eventually would be nice to display what we can without connecting the wallet, including the TWAP!